tests/testthat/_snaps/create_registry/registry.json

```diff
@@ -86,6 +123,7 @@
       "onboarding": "",
       "on_cran": false,
       "on_bioc": false,
+      "type": "experimental",
```

Comment:
a good side effect of snapshot testing is that the diff of this snap shows what I've added
* the type field
* ropensci-archive repos (well in the tests)